engine/graphics/src/wscript

```diff
@@ -27,6 +27,15 @@ def build(bld):
                     use             = 'DDF DLIB',
                     target          = 'graphics_proto')
 
+    bld.add_group()
+
+    obj = bld.stlib(features        = obj.features + ['skip_asan'],
+                    includes        = obj.includes,
+                    source          = bld.path.ant_glob(['graphics_proto.cpp']) + bld.path.parent.ant_glob('proto/graphics/*'),
+                    protoc_includes = obj.protoc_includes,
+                    use             = obj.use,
+                    target          = 'graphics_proto_noasan')
```

Comment:
Particle now depends on graphics being built first, since we use data structures from that library